isofit/radiative_transfer/radiative_transfer.py

```diff
@@ -99,7 +99,8 @@ def __init__(self, full_config: Config):
             # Generate the params for this RTE
             params = {
                 key: confPriority(key, [confRT, confIT, config]) for key in self._keys
-            } | {"engine_config": confRT}
```

Comment:
Python 3.11 is the first version to support merging dictionaries with the `|` operator. Use `dict.update()` for broader compatibility instead.